src/server.ts

```diff
@@ -2,16 +2,19 @@
  * Main entry point for the Digital Citizenship proxy.
  */
 
-import * as dotenv from "dotenv";
 import * as http from "http";
 import * as winston from "winston";
 import { newApp } from "./app";
+import container from "./container";
+import { CIDRString } from "./utils/strings";
 
-dotenv.config();
+const port = container.resolve<number>("serverPort");
```

Comment:
just a comment, something I don't like about the IC approach is that in this exact moment you lose type safety - if by mistake you put the wrong type here, you will introduce a but and the compiler will not be able to catch it